src/main/java/org/rpgl/json/JsonArray.java

```diff
@@ -0,0 +1,172 @@
+package org.rpgl.json;
+
+import java.io.Serial;
+import java.util.ArrayList;
+import java.util.Collection;
+
+/**
+ *
+ * @author Calvin Withun 10-16-2022
+ *
+ * <p>
+ * The <code>JsonArray</code> class represents json arrays. All
```

Comment:
Go back and expand these javadocs to the full width allowed. Do this for all javadocs left in the narrow form.